index.html

```diff
@@ -35,5 +38,137 @@ <h1 class="heading-1">
             <li><a href=""><i class="fab fa-instagram"></i></a></li>
         </ul>
     </section>
+
+    <img src="img/image_geometry_2.svg" alt="" class="image-break">
+    
+    <section class="section-2">
+        <h1 class="heading-1">Projects</h1>
+    </section>
+    <section class="section-3">
+        
+        <div class="background-color">
+            <img src="img/Rectangle21.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
```

Comment:
- [ ] The main container of the projects has a display:grid, please display the container of the project as display:grid.

Reply:
Thank you. Appreciate the feedback, issues have been handled